packages/web3-providers-ws/package.json

```diff
@@ -9,6 +9,6 @@
     "dependencies": {
         "underscore": "1.8.3",
         "web3-core-helpers": "1.0.0-beta.37",
-        "websocket": "git://github.com/frozeman/WebSocket-Node.git#browserifyCompatible"
+        "websocket": "1.0.28"
```

Comment:
The only addition of the `browserifyCompatible` branch in frozeman's fork seems to be [this change](https://github.com/frozeman/WebSocket-Node/commit/7004c39c42ac98875ab61126e5b4a925430f592c). I'm not sure why it was made in the first place, but are we sure that it is no longer needed?

Reply:
I'm using the ``websocket`` package in the new architecture directly but will definitely do some additional tests. (There was an issue in the past with getting the global object with Firefox)